### ISSUE TITLE
Jodel is not happy about this

### ISSUE THREAD

**Issue by ioncodes** [nborrmann/jodel_api#39]
Maybe you received an email as well. The Jodel Team is not happy about me doing this, so they contacted me, and I'd like to let you know, that you should also stop your work on this, until I find a solution with them.

**Comment by KLVN** [comment]
:(

**Comment by MichaelHepp** [comment]
Are they threatening you with legal issues?

**Comment by aschaeufler** [comment]
Can you maybe explain more in detail what they have said?

**Comment by ioncodes** [comment]
They told me to close all repos regarding my JodelAPI and all it's sub projects, because they were used to spam on jodel. That's pretty much the short version.

**Comment by aschaeufler** [comment]
Thank you for the fast reply. :) They should offer at least an offical read only api. :D

**Comment by nborrmann** [comment]
I don't think that deleting (what does "closing" even mean?) the repos makes any sense, there's too many forks and they won't be able to remove all of them.

They told me that they're mostly fine with read-only functions, the problem is the verify() function that I built, and I kinda agree.

I guess I will move the gcmhack stuff to a different repo and remove all the Jodel specific stuff. That should stop most script kiddies from spamming, because they'll have to decompile the app themselves to get the IDs and keys for GCM.

**Comment by ioncodes** [comment]
Closing a repo means removing it or setting it to private.

**Comment by nborrmann** [comment]
Yeah, that won't solve anything. all the forks can still be found via google or github search.

**Comment by KLVN** [comment]
They can request a [DMCA takedown notice](https://help.github.com/articles/dmca-takedown-policy/#dmca-notices-in-a-nutshell) to disable this repo and forks. But the internet never forgets and I think many of us have a local copy :P

**Comment by KLVN** [comment]
Any news on this?

We programmed a bot for Jodel using your API and asked Jodel if we can keep it running, because we have no bad intentions and it helped the community (had over 1.200 members in our channel and positive feedback). They told us that they are already thinking about a public API for developers like us and as a test (I think) they created a custom account for us, that no longer needs to be refreshed. On the other side our actions are monitored and if the conditions are violated they close our account.

It may seem strict, but we are no longer in fear, that they ban our account because we are not following the rules, because it's kind of "official" now.

So if you also have a "good" program/bot you should contact Jodel via [facebook](m.me/JodelDACH) and they give you an email adress for further instructions. And don't forget to tell them how great it would be to have an official and public API ;)

**Comment by ioncodes** [comment]
They told me all that aswell, I've been invited to Berlin and I will meet them there. I will definitely discuss this with them. I made a few plans/concepts for the API and it's security, so I hope that we will find a great solution for this.

**Comment by KLVN** [comment]
> I've been invited to Berlin and I will meet them there.

![](http://i.imgur.com/HcqORAn.gif)

**Comment by ioncodes** [comment]
😂😂😂 Wasn't meant to be like a dick 😂

**Comment by anaseqal** [comment]
@ioncodes can you send me your contacting info please

**Comment by ioncodes** [comment]
@anaseqal If you tell me why and who you are, yes.

**Comment by anaseqal** [comment]
@ioncodes sure, i am looking for Python developers who interested in reverse engineering, either remotely or local.. please let me know if you interested.

**Comment by ioncodes** [comment]
@anaseqal Sounds cool, join me on Gitter and shoot me a PM: https://gitter.im/JodelAPI/Lobby?utm_source=share-link&utm_medium=link&utm_campaign=share-link

**Comment by bebehei** [comment]
> I've been invited to Berlin and I will meet them there.

@ioncodes Any updates on here?

**Comment by cpyix** [comment]
What is the offical statement from the jodel hq? Is there a way of developing "good" bots (like a daily weather bot or some cool things) for jodel which are using the api with the agreement of jodel?

**Comment by ioncodes** [comment]
@cpyix We are allowed to use read endpoints. We need to wait for them to create a public API.

**Comment by m7ariri** [comment]
so I'm not sure does this mean this api will not be updated with the new signing key? and we have to wait until the release of an official api?
